src/AppBundle/Resources/views/Admin/dashboardIframe.html.twig

```diff
@@ -62,40 +62,6 @@
     </aside>
 </div>
 
-<div class="modal fade" id="user-modal" tabindex="-1" role="dialog" aria-labelledby="user-modal-label">
```

Comment:
Is it really better? Also, the modal has a different look'n'feel.

Reply:
I think it's better to have everything inside the "react world" because the components should be reusable anywhere + handle modal state with Redux.